projects/plugins/jetpack/modules/contact-form/js/grunion-admin.js

```diff
@@ -179,8 +179,47 @@ jQuery( function ( $ ) {
 		} );
 	} );
 
-	// Handle export
-	$( document ).on( 'click', '#jetpack-export-feedback', function ( e ) {
+	// Handle export to Google Drive
+	$( document ).on( 'click', '#jetpack-export-feedback-gdrive', function ( event ) {
+		event.preventDefault();
+		var $btn = $( event.target );
+		var nonceName = $btn.data( 'nonce-name' );
+		var nonce = $( '#' + nonceName ).attr( 'value' );
+		var date = window.location.search.match( /(\?|\&)m=(\d+)/ );
+		var post = window.location.search.match( /(\?|\&)jetpack_form_parent_id=(\d+)/ );
+
+		var selected = [];
+		$( '#posts-filter .check-column input[type=checkbox]:checked' ).each( function () {
+			selected.push( parseInt( $( this ).attr( 'value' ), 10 ) );
+		} );
+
+		$btn.attr( 'disabled', 'disabled' );
+		$.post(
+			ajaxurl,
+			{
+				action: 'grunion_export_to_gdrive',
+				year: date ? date[ 2 ].substr( 0, 4 ) : '',
+				month: date ? date[ 2 ].substr( 4, 2 ) : '',
+				post: post ? parseInt( post[ 2 ], 10 ) : 'all',
+				selected: selected,
+				[ nonceName ]: nonce,
+			},
+			function ( payload, status ) {
+				if ( status === 'success' && payload.data && payload.data.sheet_link ) {
+					window.open( payload.data.sheet_link, '_blank' );
+				}
+			}
+		)
+			.fail( function () {
+				window.alert( 'There was an error exporting your results' );
```

Comment:
We would probably want this to be translatable as well. In this case, we would probably need to use `wp_localize_script` to pass the string, like we do here:
https://github.com/Automattic/jetpack/blob/ff6dad2c4a3a6ee807c1fe799f137bb08eed66e3/projects/plugins/jetpack/modules/contact-form/admin.php#L886-L896